lib/routes/ajcass/shxyj.ts

```diff
@@ -0,0 +1,84 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/shxyj/:year?/:issue?',
+    categories: ['academic'],
```

Comment:
This category [does not exist](https://github.com/DIYgod/RSSHub/blob/0179806f52d04ee448850e38173afeadb939e3ad/lib/types.ts#L5).